content/influxdb/v2.0/get-started.md

```diff
@@ -315,6 +315,32 @@ docker run --name influxdb -p 8086:8086 quay.io/influxdb/influxdb:v2.0.4
 ```
 _To run InfluxDB in [detached mode](https://docs.docker.com/engine/reference/run/#detached-vs-foreground), include the `-d` flag in the `docker run` command._
 
+### Persist data outside the InfluxDB container
+
+1. Create a new directory in which to store your data, and change to that directory.
+   ```sh
+   mkdir my-influxdb-docker-data
+   cd my-influxdb-docker-data/
+   ```
+2. From within your new data directory,
+   run the InfluxDB Docker container with the `-v` flag as in the example below.
+   This will persist the data from `/root/.influxdb2/` _inside_ the container to `my-influxdb-docker-data/` on the host file system.
+   ```sh
+   docker run \
+       --name influxdb \
+       -p 8086:8086 \
+       -v $PWD:/root/.influxdbv2 \
+       quay.io/influxdb/influxdb:v2.0.4 && \
+   ```
```

Comment:
~~~suggestion
2. From within your new directory, run the InfluxDB Docker container with the `--volume` flag to
   persist data from `/root/.influxdb2/` _inside_ the container to the current working directory in
   the host file system.
   
   ```sh
   docker run \
       --name influxdb \
       -p 8086:8086 \
       --volume $PWD:/root/.influxdbv2 \
       quay.io/influxdb/influxdb:v2.0.4
   ```
~~~